### PR TITLE
Added document property to CheckAddressOrNull & added proxyUrl to options input in BroadbandSignupUS 

### PR DESCRIPTION
**Pull request by Xedros**

### CODE REVIEW

File: src/schema/broadband-signup-us.json [hunk 1]
```diff
@@ -268,8 +272,8 @@
         "CreditCheckOrNull": {
             "oneOf": [{
                     "type": "null"
-                },
-                {
+                    },
```
Comment (randunel) [inline src/schema/broadband-signup-us.json:275]: This change is not necessary, leads to the wrong indentation. Same for the next line.

File: src/schema/broadband-signup-us.json [hunk 2]
```diff
@@ -281,6 +285,107 @@
...
                     },
+                    "document": {
+                        "type": "object",
+                        "description": "Person ID (passport or other document). Automation may fail if this information is required by the website, but not provided.",
+                        "pii": true,
+                        "properties": {
+                            "type": {
+                                "type": "string",
+                                "description": "Document type.",
+                                "enum": [
```
Comment (randunel) [inline src/schema/broadband-signup-us.json:296]: Are all of these supported by the supplier?

Comment (Xedros) [reply src/schema/broadband-signup-us.json:296]: Yes.

File: src/schema/broadband-signup-us.json (hunk 1 above)
Comment (Xedros) [reply src/schema/broadband-signup-us.json]: Yeah...the formatter from VSC...

**Pull request merged**